pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountHLLAggregationFunction.java

```diff
@@ -67,7 +79,7 @@ public void aggregate(int length, AggregationResultHolder aggregationResultHolde
     DataType valueType = blockValSet.getValueType();
 
     if (valueType != DataType.BYTES) {
-      HyperLogLog hyperLogLog = getDefaultHyperLogLog(aggregationResultHolder);
+      HyperLogLog hyperLogLog = getDefaultHyperLogLog(aggregationResultHolder, _log2M);
```

Comment:
You don't need to pass member variable `_log2M` to `getDefaultHyperLogLog()`, just remove the `static` should be good